app/models/file.ts

```diff
@@ -185,151 +185,6 @@ export default class FileModel extends ModelBaseMixin {
     return this.scanProgressClass(isStaticDone);
   }
 
-  get isNoneStatus() {
-    const status = this.dynamicStatus;
```

Comment:
Is `dynamnicStatus` property in this model still required? I don't think it was deleted.